client/src/pages/Settings/Settings.tsx

```diff
@@ -37,6 +38,11 @@ const settingsMenu = [
     to: '/profile/settings/billings',
     component: <StripeConnect />,
   },
+  {
+    name: 'Custom booking',
+    to: '/custom/booking',
+    component: <CustomBookingList header="Custom Booking" />,
+  },
```

Comment:
Not sure why this is on the settings page? This should be a page in the navbar for the customer.

Reply:
Make sure to move the components out of the settings folder to the pages